apps/web/pages/apps/categories/[category].tsx

```diff
@@ -47,7 +47,7 @@ export default function Apps({ apps }: PageProps) {
         <div className="mb-16">
           <div className="grid-col-1 grid grid-cols-1 gap-3 md:grid-cols-3">
             {apps
-              .sort((a, b) => (b.installCount || 0) - (a.installCount || 0))
+              ?.sort((a, b) => (b.installCount || 0) - (a.installCount || 0))
```

Comment:
i didn't got any error when i tried a category that doesn't exist

Reply:
Sir, I got it in local development, I send a screenshot too